content/en/docs/getting-started/application-developer.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Application Developer
+description: >-
+  I write code for my organization's apps and services. My goal is to get
```

Comment:
The only thing that I think needs changing is perhaps switching from "I" to "You". I don't know how to properly phrase it, but it just felt kinda _weird_ to me to read it as-written right now.

Reply:
I was going back and forth between both, I see good reasons for both, but let me go with "You" :-)